visualize_classification_groups.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+from PIL import Image
+import numpy as np
+from pptx import Presentation
+from pptx.util import Inches
+from io import BytesIO
+from tqdm.autonotebook import tqdm
+import matplotlib.pyplot as plt
+import tables
+
+def addimagetoslide(slide,image_stream,left,top, height, width, resize = 1.0, comment = ''):
+	# res = cv2.resize(img , None, fx=resize,fy=resize ,interpolation=cv2.INTER_CUBIC) #since the images are going to be small, we can resize them to prevent the final pptx file from being large for no reason
+	# image_stream = BytesIO()
+	# Image.fromarray(res).save(image_stream,format="PNG")
+	slide.shapes.add_picture(image_stream, left, top ,height,width)
+	txBox = slide.shapes.add_textbox(left, Inches(1), width, height)
+	tf = txBox.text_frame
+	tf.text = comment
+
+if __name__ == "__main__":
+	parser = argparse.ArgumentParser()
+	parser.add_argument('pytable_path', type=str, help='The path to an h5 file that contains "patch", "ground_truth_label", and "fname" datasets.')	
+	parser.add_argument('ppt_save', type=str, help='The full path to save the generated powerpoint.')
+
+	parser.add_argument('--str_criteria', '-s', default=["TRUE NEGATIVES", "FALSE NEGATIVES", "FALSE POSITIVES", "TRUE POSITIVES"], type=str, nargs="+")
+	parser.add_argument('--criteria', default=['00', '10', '01', '11'], type=str, nargs="+", help="Each item is a pair of ground-truth,prediction labels.")
+	args = parser.parse_args()
+	
+
+	##### INITIALIZE CRITERIA DICT #####
+	criteria_dict = {}
```

Comment:
you can actually do this entire section in 1 line : ) 

```
criteria_dict = {args.str_criteria[i]: [int(j) for j in args.criteria[i]] for i in range(len(args.str_criteria))}
```

is it more readable? perhaps not, but this works: )